src/stories/tasks/TasksTable.stories.tsx

```diff
@@ -18,21 +19,45 @@ import {
 } from '../fixtures';
 
 export default {
-  title: 'Tasks/Table/Component',
+  title: 'Tasks/Table/Example',
   component: TasksTableComponent,
   decorators: [withRedux()],
 };
 
-const Template: Story<ComponentProps<typeof TasksTableComponent>> = (args) => (
-  <TasksTableComponent {...args} />
+const sampleTasks: { [key: string]: Task } = {
+  Approved: sampleTask1,
+  Planned: sampleTask2,
+  Complete: sampleTask3,
+  'Changes Requested': sampleTask4,
+  'In Progress': sampleTask5,
+};
+
+type Props = ComponentProps<typeof TasksTableComponent>;
+
+interface StoryProps extends Omit<Props, 'tasks'> {
+  tasks: string[];
+}
+
+const Template = ({ tasks, ...rest }: StoryProps) => (
+  <TasksTableComponent tasks={tasks.map((opt) => sampleTasks[opt])} {...rest} />
 );
 
-export const Component = Template.bind({});
-Component.args = {
+export const TasksTable: Story<StoryProps> = Template.bind({});
+TasksTable.args = {
   projectSlug: 'my-project',
   epicSlug: sampleEpic1.slug,
-  tasks: [sampleTask1, sampleTask2, sampleTask3, sampleTask4, sampleTask5],
   epicUsers: [sampleGitHubUser1, sampleGitHubUser2, sampleGitHubUser3],
   openAssignEpicUsersModal: action('openAssignEpicUsersModal'),
   assignUserAction: action('assignUserAction'),
 };
+TasksTable.argTypes = {
+  tasks: {
+    defaultValue: Object.keys(sampleTasks),
+    control: {
+      type: 'multi-select',
+      options: Object.keys(sampleTasks),
+    },
+  },
```

Comment:
👍 I'm not entirely sure what's happening here but I think it's much cleaner and lets us write less stories.  reading docs [here](https://storybook.js.org/docs/react/api/argtypes) to see what this is doing...

Reply:
By default, Storybook pulls in every component `prop` as a `control`, and makes it editable in real-time via the lower "Controls" tab. Stacy pointed out that's not very useful for many of our props, which are often lists of really large objects e.g. tasks or epics. So, I disabled the control entirely in some places (like `epicUsers` below: `  epicUsers: { control: { disable: true } },`), and in other places (`TasksTable.tasks` and `EpicTable.epics`) I'm trying out this method. Basically instead of showing the raw JSON object for the list of `tasks`, we're showing a multi-select with custom names of types of tasks, by default with all of them selected. Then our Story `Template` above gets that list of args and converts it into the tasks themself. It's most similar to (and inspired by) the examples here: https://storybook.js.org/docs/react/essentials/controls#fully-custom-args

Like I said, I'm not tied to it. I wanted to try it out and see what you all think -- I'm open to however the three of you think is best and most useful.